packages/liquidator/test/index.js

```diff
@@ -418,10 +418,16 @@ contract("index.js", function(accounts) {
         // To verify decimal detection is correct for a standard feed, check the third log to see it matches expected.
         assert.isTrue(spyLogIncludes(spy, 3, `"contractVersion":"${contractVersion.contractVersion}"`));
         assert.isTrue(spyLogIncludes(spy, 3, `"contractType":"${contractVersion.contractType}"`));
-
+      });
+      it("Correctly rejects unknown contract types", async function() {
```

Comment:
I split this test into two parts to be more explicit as to what is being tested & validate the error string correctly.